docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,27 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Allow push of nondistributable artifacts
```

Comment:
@thaJeztah: I documented this in 93ca5e61c2408342495514b8c0f7a88a1de763ae. If that looks OK, I'll squash it.